Week02/lesson04/homework/relaxr_starter_code/index.html

```diff
@@ -1,27 +1,83 @@
 <!DOCTYPE html>
+
 <html lang="en">
+
 <head>
+
 	<meta charset="utf-8">
 	<meta name="description" content="Your description goes here">
 	<meta name="keywords" content="one, two, three">
-	<title>Blank Template</title>
+	<title>Relaxr</title>
 
-	<!-- external CSS links -->
 	<link rel="stylesheet" href="css/reset.css">
 	<link rel="stylesheet" href="css/style.css">
+
 </head>
 
 <body>
 
-	<div id="container">
-		<header>
-			<h1>Blank Template</h1>
-			<h2>Just another blank template</h2>
-		</header>
+<div id="container">
+
+<header>
+
+			<h1>Relaxr</h1>
+
+			<h2>Get piece of mind with a single tap</h2>
+
+				<nav>
+				<a href="#">Get it Now</a>
+			  </nav>
+
+</header>
+
+<div class="benefits">
+
+			<h3>Benefits</h3>
+
+			<br>
+
+			<p>The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
+
+			<br>
+
+			<ol>
+					<li>Schedule meetings for you</li>
+					<li>Excel automation to complete your work for you</li>
+					<li>Responds to emails on your behalf</li>
+					<li>Does all your work for you with our revolutionary AI technology</li>
+		  </ol>
+
+</div>
+
+<div class="quote">
+
+			<blockquote>"Relaxr changed my life. I’ve been able <br/>
+					         to travel the world, spend limited time <br/>
+									 working and my boss keeps thanking <br />
+									 me for crushing work.” <br/>
+                  <span/>- Amanda, Intuit<span/></blockquote>
+
+</div>
+
+<div class="link">
+
+      <nav>
+	    <a href="#">Sign Up Now!</a>
+      </nav>
+
+</div>
+
+<footer>
+
+			<h1>Relaxr</h1>
+
+			<img src="images/twitter.png">
```

Comment:
Try wrapping each of the social media images in an `<a>` next time and have them link to FB and twitter for real!

Bonus: google `target="blank"` and see how that changes the behaviour of clicking.